src/_utils.js

```diff
@@ -142,7 +142,7 @@ export const validateExpressionVisitor = {
         (t.isThisExpression(node.object) &&
           (node.property.name === 'props' ||
             node.property.name === 'state' ||
-            node.property.name === 'context'))) ||
+            node.property.name === 'registry.js'))) ||
```

Comment:
I think you should revert this

Reply:
hmm. I believe IntelliJ did this when I renamed a file. Obviously not covered by tests.